client/src/components/InputTextField/InputTextField.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import "./InputTextField.css";
+import "../../App.css";
+/**
+ * Created by Tyler
+ *
+ * Props will define the type of input field
+ * Input fields include email, password, name, confirm password, city of residence, what is your age?, what is your dream destination, First name,   last name, search my directory, search trips.
+ *
+ * I think we'll need to implement a function on these to retrieve the data typed in when pressing a submit button. I wasn't sure how to do this with just the standalone components in storybook.
+ */
+
+// Class Based React Component
+const InputTextField = props => {
+  let { size, type, name, placeholder, onChange } = props;
+  let classList = "input-text-field-container";
+  if (size === "small") {
+    classList = "input-text-field-container-small";
+  } else if (type === "search") {
+    classList = "input-text-field-container search";
+  } else if (size === "small" && type === "search") {
+    classList = "input-text-field-container-small search";
+  }
```

Comment:
I'd set a default prop value for `type` (maybe `"text"`) - that way you know you'll always have a value. Then you can modify this to just be a single line of string interpolation:

```js
let classList = `input-text-field-container intput-text-field-container-${type}`
```

If `type` is `email`, then you'll get: `"input-text-field-container intput-text-field-container-email"`

Another side note is these classnames are super long. You could go with `input-field`, `input-field-container`, `input-field-text`, `input-field-email`, `input-field-search`, etc

Reply:
Oh and better yet - don't do this with a variable, just do it on line 26 when you set the `className` for the `div`